platforms/common/src/kernels/removeCM.cc

```diff
@@ -19,22 +19,26 @@ KERNEL void calcCenterOfMassMomentum(int numAtoms, GLOBAL const mixed4* RESTRICT
 
     int thread = LOCAL_ID;
     temp[thread] = cm;
+
+    // Easier to cope with varying block / wavefront sizes w/o perf. penalty if
+    // expressed as a constexpr reduction
```

Comment:
Keep in mind that CM motion removal takes truly negligible time.  It's not worth adding any complexity to optimize it, because there will be no practical benefit.